client/src/components/publications/form/PublicationForm.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * The PublicationForm component displays a mutli-step publication form
+ */
+
+import { updatePublication, createPublication } from '../../../actions/publications'
+import React, {useState} from 'react';
+import { useDispatch } from 'react-redux';
+import PageOne from "./MainInfoPage"
+import PageTwo from "./CategoryPage"
+
+export const CATEGORY_TYPE = {
+    Journal: "JOURNAL",
+    Conference: "CONFERENCE"
+}
```

Comment:
minor: should these go in a separate `.js` file? I was thinking we can use component files to only export default the react component itself and create separate files for constants such as this.